scripts/driver/blackwidow_elite/test.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/python3
```

Comment:
Do we need this file in the tree?

Reply:
This script helps to check all keyboards features and has been written as others test scripts (for blackwidow_chroma_keyboard, ornata_chroma_keyboard and etc).